src/components/MapView/Analyser/index.tsx

```diff
@@ -448,7 +600,13 @@ function Analyser({ extent, classes }: AnalyserProps) {
                   {isTableViewOpen && (
                     <AnalysisTable
                       tableData={analysisResult.tableData}
-                      columns={getAnalysisTableColumns(analysisResult)}
+                      columns={
+                        'tableColumns' in analysisResult
+                          ? // have to add "as any" here because typescript compiler mistakenly thinks
+                            // analysisResult is a union of BaselineLayerResult and PolygonAnalysisResult
+                            (analysisResult as any).tableColumns
```

Comment:
And what is it instead? Could you cast to the actual type?

Reply:
Thanks for catching this.  Fixed with https://github.com/WFP-VAM/prism-frontend/pull/431/commits/40303b2f38eb134d92ce99d1ceb0add53fe279f7